packages/db/index.ts

```diff
@@ -1,14 +1,33 @@
 import { OPENAI_API_KEY } from "@repo/ai/constants";
-import { DB_HOST, DB_USER, DB_PASSWORD, DB_NAME } from "@repo/db/constants";
+import { DB_HOST, DB_USER, DB_PASSWORD, DB_NAME, DB_PORT, TIER } from "@repo/db/constants";
 import { createEleganceServerClient } from "@singlestore/elegance-sdk/server";
 
+let cert;
+
+fetch("https://portal.singlestore.com/static/ca/singlestore_bundle.pem")
+  .then(function (response) {
+    response.text().then(function (text) {
+      cert = text;
+    });
+  })
```

Comment:
Let's move this logic into the `if (TIER === "shared") {` block below to fetch a certificate only if necessary to prevent fetching on every page load, even if not required for a regular workspace.

Reply:
done! did this for all other repos as well